test-app/tests/integration/components/field-test.gts

```diff
@@ -96,4 +96,28 @@ module('Integration | Component | Field', function (hooks) {
     assert.dom('[data-test-hintId]').hasAnyText();
     assert.dom('[data-test-errorId]').hasAnyText();
   });
+
+  test('it renders an array of errors', async function (assert) {
+    let testErrors = ['error 1', 'error 2'];
```

Comment:
Suggestion, non-blocking (maybe typescript will catch this anyhow??): can we test for an array of numbers or a not-string/not-array-of-string condition?

Reply:
Also: maybe we should test for an empty array here, to confirm that no errors are displayed.

Reply:
Yeah actually now that I'm thinking about it, I think I have a bug here as an empty array I think will render the error blocks 😬. That's what I get for dropping caffeine and writing code in the AM 😂. Thank you for catching this! Both sound great - I'll add an assertion for the strings of the array as well tomorrow AM!